### PR TITLE
Bump InfluxDB-Watcher to include read bug fix

### PR DESCRIPTION
**Pull request by aadams19**
Pulls in https://github.com/monasca/monasca-watchers/pull/12

Signed-off-by: Andrea Adams aadams@hpe.com

### CODE REVIEW

File: influxdb-watcher/build.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 repository: monasca/influxdb-watcher
 variants:
-  - tag: 0.0.1
+  - tag: ':latest'
```
Comment (timothyb89) [inline influxdb-watcher/build.yml]: whoops, there shouldn't be a colon here

File: influxdb-watcher/build.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
     aliases:
-      - ':latest'                     
+      - 0.0.2
```
Comment (timothyb89) [inline influxdb-watcher/build.yml]: and we do need a colon here

**Comment by timothyb89** [comment]
it's a bit concerning that CI didn't catch that... I would expect an error...

File: influxdb-watcher/build.yml (hunk 1 above)
Comment (aadams19) [reply influxdb-watcher/build.yml]: Done

File: influxdb-watcher/build.yml (hunk 2 above)
Comment (aadams19) [reply influxdb-watcher/build.yml]: Lol, totally mixed them up

**Comment by timothyb89** [comment]
oh!

```
Module influxdb-watcher does not belong to metrics, skipping
```

we need to add influxdb-watcher to the metrics pipeline: https://github.com/monasca/monasca-docker/blob/master/ci.py#L42

that can be a separate commit, as long as we actually publish something here

**Pull request merged**